bittensor/extrinsics/network.py

```diff
@@ -210,3 +210,109 @@ def set_hyperparameter_extrinsic(
                     f":white_heavy_check_mark: [green]Hyper parameter {parameter} changed to {value}[/green]"
                 )
                 return True
+
```

Comment:
Why have these files changed ? I do not think they do not have anything to do with the pr and should be in the diff

Reply:
To register a subnetwork, there is a network_rate_limit. the default value is 7200. I add an extrinsic to set it. then we can register soon after network is up.